tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -971,8 +976,9 @@ protected:
          fSubFields[0]->Append(itemValue);
       }
       Detail::RColumnElement<ClusterSize_t, EColumnType::kIndex> elemIndex(&fNWritten);
-      fNWritten += count;
       fColumns[0]->Append(elemIndex);
+      printf("just appended: %d\n", fNWritten);
+      fNWritten += count;
```

Comment:
Append to the offset column before adding count, e.g. 
Before: `[3 5 7]`
After:    `[0 3 5]`
and we append 7 in `CommitCluster`